extension/src/config/config.js

```diff
@@ -26,7 +26,8 @@ function getAdyenCredentials () {
     merchantAccount: process.env.ADYEN_MERCHANT_ACCOUNT,
     apiKey: process.env.ADYEN_API_KEY,
     apiBaseUrl: process.env.ADYEN_API_BASE_URL || 'https://checkout-test.adyen.com/v52',
```

Comment:
should we update to `v65`?

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/469/commits/a49e2fe985913df1e01b73018f6e2216489c9cbc